README.md

```diff
@@ -402,6 +402,35 @@ cd aws-android-sdk-appsync-tests/
 
 To run tests from Android Studio, run `./gradlew publishToMavenLocal` from project root, then open the `aws-android-sdk-appsync-tests/` directory in Android Studio, and run the tests through the UI.
 
+### Integration testing infrastructure
```

Comment:
Since this is an unlikely branch for the average reader, should we put it into a `CONTRIBUTING.md`? Or, maybe a `TESTING.md`?

Reply:
Ah...that's a good point. It felt kind of out of place. I'll move it to an separate file.